.github/workflows/build_and_push.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and Push Docker Image
```

Comment:
Is this workflow only related to development images?

Reply:
No, just a workflow to create and push image manually from whatever branch.